File: Pattern35.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Enter the number of rows: ", (n) => {
  for (let i = 1; i <= n; i++) {
    for (let space = n; space >= i; space--) {
      process.stdout.write(" ");
    }
    process.stdout.write("*");
    if (i === 1 || i === Number(n)) {
      for (let j = 1; j < n; j++) process.stdout.write("*");
    } else {
      for (let k = 2; k < n; k++) process.stdout.write(" ");
      process.stdout.write("*");
    }

    process.stdout.write("\n");
  }

  rl.close();
});

File: PATTERNS/Pattern35/Pattern35.py
n = int(input("Enter the number:"))
for i in range(1, n+1):
    print(" "*(n-i), end="")
    if i == 1 or i == n: #First and last line
        print("*"*n)
    else:
        print("*"+" "*(n-2)+"*")